application/deed/deed-api.json

```diff
@@ -1,153 +1,195 @@
-                "summary": "Deed",
-                "description": "The Deed endpoint returns details of a specific deed based on the unique deed reference.\nThe response includes the Title Number, Property information, Borrower(s) information and deed information.\n",
-                "produces": [
-                    "application/json"
-                ],
-                "parameters": [
-                    {
-                        "name": "deed_reference",
-                        "in": "path",
-                        "description": "Unique reference of the deed.",
-                        "required": true,
-                        "type": "string"
-                    }
-                ],
-                "tags": [
-                    "Deeds"
-                ],
-                "responses": {
-                    "200": {
-                        "description": "A specific deed is returned",
-                        "schema": {
-                            "$ref": "#/definitions/Operative_Deed"
-                        }
-                    },
-                    "404": {
-                        "description": "Deed not found",
-                        "schema": {
-                            "$ref": "#/definitions/Error"
-                        }
-                    }
-                }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "URL to the GET endpoint for the deed is returned on successful creation.\n"
+          },
+          "400": {
+            "description": "Bad Request due to invalid schema. Response will include 1 or more schema errors\n",
+            "schema": {
+              "$ref": "#/definitions/Error"
             }
+          }
         }
+      }
     },
-    "definitions": {
-        "Deed_Application": {
-            "type": "object",
-            "properties": {
-                "title_number": {
-                    "type": "string",
-                    "description": "Unique Land Registry identifier for the registered estate."
-                },
-                "borrowers": {
-                    "type": "array",
-                    "description": "A list of borrowers.",
-                    "items": {
-                        "$ref": "#/definitions/Borrower"
-                    }
-                }
+    "/deed/{deed_reference}": {
+      "get": {
+        "summary": "Deed",
+        "description": "The Deed endpoint returns details of a specific deed based on the unique deed reference.\nThe response includes the Title Number, Property information, Borrower(s) information and deed information.\n",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deed_reference",
+            "in": "path",
+            "description": "Unique reference of the deed.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "Deeds"
```

Comment:
Good job Dan. It is deed, saw it, wasn't overly fussed by it but it is a quick fix if you want to Ramin.